src/amber/router/router.cr

```diff
@@ -58,24 +55,7 @@ module Amber
         @routes_hash["#{controller}##{action}"]
       end
 
-      def all
```

Comment:
This is fine I added those methods to for the `amber routes` command so we can get the routes that have been register with the app. It turns out that we are currently parsing the routes.cr file which does not tell the full truth since routes can be register via code.